doc/sphinxext/gen_rst.py

```diff
@@ -1,173 +0,0 @@
-"""
```

Comment:
I am concerned that someone is depending on this.  There is not much harm in carrying this file.

Reply:
On second thought, was this even importable outside of our docs build? :sheep:

Reply:
The importable stuff is in `lib/matplotlib/sphinxext`.